ui/modModules/multiplayer/multiplayer.css

```
@@ -420,3 +423,67 @@ md-slider.ng-pristine.ng-untouched.ng-valid.md-default-theme.ng-empty {
   top: 2px;
   max-width: 34px;
 }
+
+
+.server-maxplayers {
+  width: 90px; 
+}
+.server-mapname {
+  width: 20%; 
+}
+.server-servername {
+  width: 70%; 
+}
+.server-flag {
+  width: 90px; 
+}
```

```suggestion
.server-maxplayers {
  width: 8%; 
}
.server-mapname {
  width: 20%; 
}
.server-servername {
  width: 64%; 
}
.server-flag {
  width: 8%; 
}
```
Final review, this one is my bad. The change I'm reverting made it initially look better but ended up causing issues with the details window. With percentages it can look a little funny on wide monitors but this is something that doesn't seem very simple to fix.

---

I can do different styles depending on the resolution with some media queries for each resolution. It takes time, and I don't really know how it needs to look at very wide resolutions, so if you have a little time to send the style needed for each resolution that I need to fix, I would be very grateful! Otherwise, I can just commit this, but I feel like it's probably too imperfect since a lot of people have a widescreen to play.

In theory, I can also change the rows height depending on the resolution. (That mean being able to change the font size etc..)

---

Alright, the flag and player count should both be 90px, the server name should take up 2/3 of the remaining space and the map 1/3.